test/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoaderTest.java

```diff
@@ -3710,5 +3712,24 @@ private IInternalContest fullLoadSampleContest(String sampleName) throws Excepti
         return contest;
     }
     
+    /**
+     * Find/match problem in contest by problem letter
+     * @param inContest
+     * @param letter
+     * @return null if not found, else problem that is found to match letter
+     */
+    // TODO REFACTOR proomote/move getProblemByLetter into AbstractTestCase
+    private Problem getProblemByLetter(IInternalContest inContest, String letter) {
+        
+        Problem[] problems = inContest.getProblems();
+        for (Problem problem : problems) {
+            if (letter.equalsIgnoreCase(problem.getLetter())) {
+                return problem;
+            }
+            
+        }
+        return null;
+    }
+    
```

Comment:
The above block of code doesn't look like it has anything to do with i295.  I'm ok with it; it's just sort of "out of scope" (unless I'm missing something....)

Reply:
This method is directly used (see above) to test testLoadSandboxAndMemoryLimit.
It is used to find a problem in the model by problem letter.   It is not extraneous,
is a helper method for the testLoadSandboxAndMemoryLimit.

Reply:
Aha; sorry, I missed that above.  Emily Litella...